src/backend/utils/misc/queryjumble.c

```diff
@@ -597,6 +598,36 @@ JumbleExpr(JumbleState *jstate, Node *node)
 				JumbleExpr(jstate, rtfunc->funcexpr);
 			}
 			break;
+		case T_GroupId:
+			/*
+			 * Struct GroupId has only the node tag, already added to the jumble
+			 * in the code above. So just leave case empty to suppress warning.
+			 */
+			break;
+		case T_Integer:
+			{
```

Comment:
The braces are unnecessary here because no variables are created on the stack.

Reply:
fixed